src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableList_1.cs

```diff
@@ -182,23 +147,14 @@ public int Count
         /// See <see cref="ICollection"/>.
         /// </summary>
         [DebuggerBrowsable(DebuggerBrowsableState.Never)]
-        object ICollection.SyncRoot
-        {
-            get { return this; }
-        }
+        object ICollection.SyncRoot => this;
 
         /// <summary>
         /// See the <see cref="ICollection"/> interface.
         /// </summary>
         [DebuggerBrowsable(DebuggerBrowsableState.Never)]
-        bool ICollection.IsSynchronized
-        {
-            get
-            {
-                // This is immutable, so it is always thread-safe.
-                return true;
-            }
-        }
+        // This is immutable, so it is always thread-safe.
+        bool ICollection.IsSynchronized => true;
```

Comment:
Does this change not produce a warning?  I think we should either leave this as a regular property getter, or move this single line comment into an XML doc comment perhaps within a devremarks tag.